### ISSUE TITLE
Change color of the AI shell version to be higher contrast

### ISSUE THREAD

**Issue by StevenBucher98** [PowerShell/AIShell#322]
### Summary of the new feature / enhancement

We need to change the version color during start up from its current color to #7A7A7A. This will make the contrast higher and more acceptable for low vision users.

![Image](https://github.com/user-attachments/assets/dc11252f-5a5f-4098-974e-e6f3265e4348)

**Comment by daxian-dbw** [comment]
Where does the color `#7a7a7a` come from? We currently use `#808080`, and they are very close to each other. I cannot notice any differences on Windows Terminal when using them in the code:

_1st - current (#808080), 2nd - #7a7a7a_

![Image](https://github.com/user-attachments/assets/d7a47416-3baf-4c3e-a8c3-c8606f69a3db)
![Image](https://github.com/user-attachments/assets/2e616c22-ca7c-479a-a00e-2b27b6fdacf3)

**Comment by StevenBucher98** [comment]
This comes from the[ accessibility insights for windows,](https://accessibilityinsights.io/docs/windows/overview/) it seemed to be the most similar to what we already have but high enough contrast to pass the contrast checker.
